app/models/health/agency_performance.rb

```diff
@@ -18,6 +18,18 @@ def self.url
       'warehouse_reports/health/agency_performance'
     end
 
+    DESCRIPTIONS = {
+      without_required_qa: 'Patients who have not completed intake and have not received a QA in the specified date range.',
+      without_required_f2f_visit: 'Patients who have not received a face-to-face visit in the two months preceding the specified end date.',
+      with_discharge_followup_completed: 'Number of discharge follow-up QAs within the specified data range.',
+      with_completed_intake: 'Patients with completed initial intake (Consent, Comp Assessment, HRSN, and Care Plan) as of the specified end date.',
+      initial_intake_due: 'Patients who need to receive an initial intake within 30 days of the specified end date.',
+      initial_intake_overdue: 'Patients who did not complete an initial intake within 153 days of their enrollment as of the earlier of the current date or the specified end date.',
```

Comment:
do we want some of these dates to come from other constants?  It's kinda weird the requirement of 153 days.  I assume we'll use that elsewhere, might be smart to force them to match.

Reply:
Good idea... I'll look at that